Sources/HttpServer/Data/DataStorage.swift

```diff
@@ -3,13 +3,28 @@ import Foundation
 public class DataStorage {
 
     private var data: [String: String]
+    private var cookieJar: [String: [String]]
```

Comment:
What is a `cookieJar`?

Reply:
It's a place to put all the cookies :) I must have thought of chocolate chip cookies when I name this variable LOL